### PR TITLE
Species column added to table

### PR DESCRIPTION
**Pull request by shegy28**
**In this PR, I was able to do the following:**

- [x] Add column species of type string to animals table.
- [x] Insert animal data into the table.
- [x] Set species to unspecified.
- [x] Set species to digimon or pokemon.
- [x] Delete all records.
- [x] Delete some records and update others.
- [x] Queries with aggregate functions.

**Transactions**
Screenshot (118)

Screenshot (119)

Screenshot (120)
Screenshot (123)

**Aggregate Functions**

Screenshot (124)

Screenshot (125)

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -10,4 +10,30 @@ INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
...
+/*Day 2 of Populate database with sample data. */
+
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) 
+VALUES ('Charmander', '2020-02-08', -11, 'false', 0); 
+
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) 
+VALUES ('Plantmon', '2021-11-15', -5.7, 'true', 2); 
+
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) 
+VALUES ('Squirtle', '1993-04-02', -12, 'false', 3);
```
Comment (danmainah) [inline data.sql]: - [ ] @shegy28 kindly ensure your figures are correct , your weight is suppose to be  -12.13kg

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @shegy28 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR and Description


# Required Changes ♻️

- [ ] @shegy28 kindly ensure your issues link is working in your readme file, also you can change live demo link to coming soon since you do not have a live demo currently

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: data.sql (hunk 1 above)
Comment (shegy28) [reply data.sql]: okay i will update it now

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @shegy28👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



 ## Required Changes ♻️

- [x] As requested by the previous reviewer, Please make sure the issues link is working in the README file, also you can change the live demo link to coming soon since you do not have a live demo currently

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @shegy28 , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- Good commit messages. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**